src/components/ReviewTransaction.tsx

```diff
@@ -190,6 +196,93 @@ export function ReviewFooter(props: { children: ReactNode }) {
   return <View style={styles.reviewFooter}>{props.children}</View>
 }
 
+export function ReviewTotalValue({
+  tokenInfo,
+  feeTokenInfo,
+  tokenAmount,
+  localAmount,
+  tokenFeeAmount,
+  localFeeAmount,
+}: {
+  tokenInfo: TokenBalance | undefined
+  feeTokenInfo: TokenBalance | undefined
+  tokenAmount: BigNumber | null
+  localAmount: BigNumber | null
+  tokenFeeAmount: BigNumber | undefined
+  localFeeAmount: BigNumber | null
```

Comment:
is there any advantage to passing in the `localAmount` and `localFeeAmount`? it seems that these can be derived using `tokenAmount` and `tokenInfo` / `feeAmount` and `feeTokenInfo`.

if the line of thinking is that these local amounts need be calculated when rendering the other details lines too, therefore we should try to calculate them once on the consumer and pass them down....then we could also have localCurrencySymbol passed in and avoid checking `!!tokenInfo.priceUsd && !!feeTokenInfo.priceUsd` in `haveLocalPrice`?